src/NuGetPackageVerifier/CompositeRules/PackageDependenciesCompositeRule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class PackageDependenciesCompositeRule : IPackageVerifierRule
```

Comment:
@Eilon @pranavkm Does `Target Name="NuGetVerify"` in the [dnx.msbuild file](https://github.com/aspnet/Coherence-Signed/blob/dev/dnx.msbuild#L172) run all the rules in `NuGetPackageVerifier`? Since we only call this once after rebuilding packages and creating timestamp free packages, why not have this new pre-rel package rule in the `DefaultCompositeRule`? If this is not what's happening, how/where in the msbuild file can one specify which composite rule to run? And in that case I think we will need more than one execution of `NuGetVerify`.